publish/update_information.json

```diff
@@ -1,7 +1,7 @@
 {
   "chrome": {
-    "latest_version":"1.0.0",
-    "url":"https://github.com/hypertrons/hypertrons-crx/releases"
+    "latest_version":"0.1.2",
```

Comment:
I think we can reserve `1.0.0` as the latest version.

Reply:
ok

Reply:
I have submited `1.0.0` version to chrome app store. I suppose it can be published today.

Reply:
> I have submited `1.0.0` version to chrome app store. I suppose it can be published today.

I have changed version to `1.0.0`